README.md

```diff
@@ -68,6 +68,25 @@ must be applied via [patches](patches). See [below](#patches) for more info on p
 1. After the PR is merged to update a component, coordination is often needed in the darc dependency flows. The source-build-external update
 may need to flow in at the same time as the cooresponding changes in product repos which take a dependency on the new component version.
 
+### Updating an External Component Used in a Toolset Repo and in a Target Repo
+
+When updating a component that is used in a repository which is built before source-build-externals during the product build (toolset repos) and is used in a repo that is built after source-build-externals (target repos), please adhere to the following steps:
```

Comment:
`toolset repo` means something different to me.  Arcade and SBRP have been referred to as toolset repos in the past.  They are the repos treated specially in the product source-build as they use the N-1 toolset to build.

Reply:
Maybe `Pre-SBE repos` is a better name then?